bower.json

```diff
@@ -68,6 +68,7 @@
     "summernote-0-8-2": "summernote#0.8.2",
     "underscore-1-8-3": "underscore#1.8.3",
     "undone.js-0-0-1": "https://github.com/yckart/undone.js.git#0.0.1",
-    "validate.js-0-10-0": "validate.js#0.10.0"
+    "validate.js-0-10-0": "validate.js#0.10.0",
+    "bootstrap-sidebar": "bootstrap-sidebar#0.2.2"
```

Comment:
note this should be:
``````
"bootstrap-sidebar-0-2-2": "bootstrap-sidebar#0.2.2"
``````
(and to keep it organized, best to sort it on this list by alphabet)
So when adding a new asset, we directly modify this file and then run bower install. see here for more details (note Scott even included a cool mechanism from composer to yank out stuff, which is not pertinent for this asset):
http://www.open-emr.org/wiki/index.php/Bower#Bower_Method_.28preferred.29